src/components/ColumnAdjuster/constants.ts

```diff
@@ -0,0 +1,18 @@
+export enum ColumnWidthValues {
+  PixelsMin = 30,
+  PixelsMax = 7680,
+  PixelsDefault = 200,
+  PercentageMin = 1,
+  PercentageMax = 100,
+  PercentageDefault = 20,
+  AutoMin = 80,
+}
+
+export const CELL_PADDING = 4;
```

Comment:
Are these intended to be used only in this component or shared and consumed in the tables?

Reply:
well, they are used all over the place. But I only need them in the styles file in this case. We could have them in the common constants file, since we have a bunch of other sizes in there?

Reply:
I moved it into the styles file for now